sebastian-rendon/tarea-2/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Mock Magazine Cover about Oscar Piastri"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Magazine</title>
+  </head>
+  <body>
+    <main class="main__magazine">
+      <span
+        title="Edition: September 4, 2022 - 42th Edition"
+        class="edition__details"
+      >
+        Sept. 4, 2022 - Ed. 42
+      </span>
+      <header class="header">
+        <h1 title="Checkered flag" class="header__magazine-title">
+          <span>checkered</span>
+          <span>flag</span>
+        </h1>
+      </header>
+      <img
+        src="./assets/img/oscar_piastri.webp"
+        title="Oscar Piastri"
+        class="img__magazine-cover"
+        alt="Formula driver Oscar Piastri holding a helmet in a celebration pose"
+        aria-describedby="main-article"
+      />
+      <section class="section__grid">
+        <article id="main-article" class="article__main-article">
+          <header>
+            <h1>Oscar Piastri</h1>
+            <h2>Who is the new McLaren driver?</h2>
+          </header>
+          <p>
+            The Australian promise talks about his expectations with the
+            Woking-based team.
+          </p>
+        </article>
+        <aside class="aside__supporting-articles">
+          <h1 class="title__supporting-article--first">
```

Comment:
@s-rendong , there should be only one h1 tag in a document

Reply:
fixed with [[8393c42](https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180)](https://github.com/emolinaab/bootcamp-001/pull/88/commits/8393c4295e96ea5b6bb1fd2804e2903da42f3180)